udata/features/territories/commands.py

```diff
@@ -12,12 +12,17 @@
 
 import requests
 
-from udata.models import (
-    Dataset, TERRITORY_DATASETS, ResourceBasedTerritoryDataset
-)
+from udata.app import create_app
+from udata.models import Dataset, ResourceBasedTerritoryDataset
 from udata.commands import submanager
 from udata.core.storages import logos, references, tmp
 
+
+app = create_app()
```

Comment:
@noirbizarre, can you see a better way to get the app (using `standalone()` generates an _AssertionError_)